packages/analytics/RNFBAnalytics.podspec

```diff
@@ -22,8 +17,24 @@ Pod::Spec.new do |s|
   s.social_media_url    = 'http://twitter.com/invertaseio'
   s.ios.deployment_target = "9.0"
   s.source_files        = 'ios/**/*.{h,m}'
+
+  # React Native dependencies
   s.dependency          'React'
-  s.dependency          'Firebase/Core', firebase_sdk_version
   s.dependency          'RNFBApp'
-  s.static_framework    = false
+
+  if defined?($FirebaseSDKVersion)
+    Pod::UI.puts "#{s.name}: Using user specified Firebase SDK version '#{$FirebaseSDKVersion}'"
+    firebase_sdk_version = $FirebaseSDKVersion
+  end
+
+  # Firebase dependencies
+  s.dependency          'Firebase/Core', firebase_sdk_version
```

Comment:
I could be wrong, but I think with the up to date Pods you can evict this dependency and it will be brought in transitively when needed ?

Reply:
You're right as RNFBApp brings this in of which all are dependant on - I originally left this in so its visually explicit event though it doesn't actually get added as it's already a dependency elsewhere.

Could just remove it though now I guess :D

Reply:
I think I'll go for removing these for the v7 release, have made a note